src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
this file is fine, but if you would like to learn more about adding a global styles reset file here is an example of styling that can be useful: https://meyerweb.com/eric/tools/css/reset/

:)